api/types.go

```diff
@@ -111,6 +111,18 @@ type Message struct {
 	Images  []ImageData `json:"images,omitempty"`
 }
 
+// CompletionProbability is returned by llama.cpp if n_probs is set.
+type CompletionProbability struct {
+	Content string            `json:"content"`
+	Probs   []CompletionProbs `json:"probs"`
+}
+
+// CompletionProbs is a tuple of a human readable next token and the according probability.
+type CompletionProbs struct {
+	Prob    float64 `json:"prob"`
+	Content string  `json:"content"`
```

Comment:
First, thanks for the PR, it's a great addition to Ollama!
It looks like `content` is always empty and you should use `tok_str` instead (see https://github.com/ollama/ollama/blob/main/llm/ext_server/utils.hpp#L647).
```suggestion
	Content string  `json:"tok_str"`
```